tidb/run

```diff
@@ -0,0 +1,6 @@
+lein run test \
```

Comment:
IMO, you can rename run to run.sh first, then refactor the running scripts later.